src/config/global-config.c

```diff
@@ -654,6 +657,80 @@ int Read_Global(XML_NODE node, void *configp, void *mailp)
                 }
 
             }
+        }
+        /*Agent's disconnection time parameters */
+        else if (strcmp(node[i]->element, xml_agents_disconnection_time) == 0) {
+            if (Config) {
+                char c;
+
+                switch (sscanf(node[i]->content, "%d%c", &Config->agents_disconnection_time, &c)) {
```

Comment:
Shouldn't we implement this time conversion logic in a separate method? We are repeating the same code within the same method.
Ideally, this should be a logic available in a helper method for all the Wazuh daemons.

Reply:
That method already existed!
Now both settings use **w_parse_time()**